### PR TITLE
Bicaws 365 implement security scanning

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: shared_terraform/shared_account_sandbox_infra_ci/main.tf [hunk 1]
```diff
@@ -31,6 +31,13 @@ module "codebuild_base_resources" {
   name            = module.label.name
   tags            = module.label.tags
   slack_webhook   = var.slack_webhook
+
+  allow_accounts = [
+    data.aws_caller_identity.current.account_id,
+    "454061736096",
```
Comment (satvinder-hullait) [inline shared_terraform/shared_account_sandbox_infra_ci/main.tf:37]: can you add these account id's as vars? and maybe use a merge to bring them together?

Comment (brettminnie) [reply shared_terraform/shared_account_sandbox_infra_ci/main.tf:37]: No need, these are static.

File: terraform_modules/was_ecs_task_https/README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@
 | fargate\_cpu | The number of cpu units | `number` | n/a | yes |
 | fargate\_memory | The amount of memory that will be given to fargate in Megabytes | `number` | n/a | yes |
 | ldap\_password\_arn | The arn of our password parameter | `string` | n/a | yes |
+| logging\_bucket\_name | TThe default logging bucket for lb access logs | `string` | n/a | yes |
```
Comment (bjpirt) [inline terraform_modules/was_ecs_task_https/README.md:31]: Typo

**Review by bjpirt — APPROVED** [review]

File: terraform_modules/was_ecs_task_https/README.md (hunk 2 above)
Comment (brettminnie) [reply terraform_modules/was_ecs_task_https/README.md:31]: I'll fix in the new pr

**Pull request merged**